pkg/api/v2/reporter/reporter.go

```diff
@@ -277,9 +277,8 @@ func attemptUpdateStatus(ctx context.Context, client ReporterResourceClient, res
 		//    Also, the status is accurate for the resource as it's stored in Gloo's memory in the interim.
 		//    This is explained further here: https://github.com/solo-io/solo-kit/pull/360#discussion_r433397163
 		if inputResourceFromRead, ok := resourceFromRead.(resources.InputResource); ok {
-			status := resourceToWrite.GetStatus()
 			resourceToWrite = inputResourceFromRead
-			resourceToWrite.SetStatus(status)
+			resourceToWrite.AddToReporterStatus(status)
```

Comment:
Do we want to call SetStatus as well here?

Reply:
(resolving outdated)